easybuild/easyconfigs/j/JAX/JAX-0.1.77-fosscuda-2019b-Python-3.7.4.eb

```
@@ -0,0 +1,55 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'ConfigureMake'
+
+name = 'JAX'
+version = '0.1.77'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://github.com/google/jax"
+description = """JAX is Autograd and XLA, brought together for high-performance machine learning research."""
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+
+builddependencies = [('Bazel', '2.0.0')]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('cuDNN', '7.6.4.38'),
+]
+
+source_urls = ['https://github.com/google/jax/archive']
+sources = ['jax-v%(version)s.tar.gz']
+checksums = ['18fa3bcebf4b1a526349db5a76a508784ec5d05324c192c40f240ccbf9480e26']
+skipsteps = ['configure', 'build']
+
+install_cmd = 'export TF_CUDA_PATHS=${EBROOTCUDA} && '
```

```suggestion
install_cmd += 'export GCC_HOST_COMPILER_PREFIX=$EBROOTBINUTILS/bin && '
```
Suggestion from @surak